src/features/headline/components/Headline.css

```diff
@@ -7,10 +7,15 @@
 }
 
 .stockInfo__list {
-  list-style-type: none;
   padding-top: 8px;
 }
 
+.stockInfo__list ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
```

Comment:
Q: why add padding and margin 0, should it default by 0?

Reply:
By default, a `<ul>` element has a property of `margin-left: 40px`, to allow for space for the bullet points. A CSS reset could be used, however I'm just defining those manually in each class